pkg/pillar/base/logobjecttypes.go

```diff
@@ -0,0 +1,443 @@
+// Copyright (c) 2020 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package base
+
+import (
+	"encoding/json"
+	"github.com/satori/go.uuid"
+	log "github.com/sirupsen/logrus"
+)
+
+// ObjectType : Predefined object types
+type ObjectType string
+
+// The following is NOT an extensive list.
+// Intent is to create a reference list for engineers to get an idea of its usage.
```

Comment:
This comment belongs to LogObjectType and not to ObjectType

Reply:
will move the comment.